implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -125,6 +125,9 @@ default Map<String, String> defaultDimensions() {
      * @since 1.8.0
      */
     default boolean enrichWithDynatraceMetadata() {
+        if (apiVersion() == V1) {
+            return false;
+        }
```

Comment:
This is just because it's unsupported by the v1 exporter right? (not related to metadata like units and descriptions, but entity metadata and such?)

Should we also add a docstring that this setting has no effect on V1?

Reply:
When I checked, the v1 exporter always exports them (but the mechanism there is different). It creates a custom device first, sets the values there, and exports that. Then only values are sent thereafter. I think I could extend it to v1 as well, but since v1 is legacy I did not want to change stuff there